src/SearchFilters.js

```diff
@@ -35,29 +35,53 @@ export let FiltersBox = withTheme(({ theme: { Box }, ...props }) => (
 ))
 FiltersBox.displayName = 'FiltersBox'
 
-let BasicSearchFilters = ({ setMode, trees, children, BasicFilters }) => (
-  <div>
-    <Flex alignItems="center" justifyContent="space-between">
-      <Flex alignItems="center">
-        <h1>Filters</h1>
-        <ToggleFiltersButton onClick={() => setMode('resultsOnly')} />
-      </Flex>
-      <div>
-        <TreePauseButton children={children} />
-      </div>
-    </Flex>
-    <LabelledList list={trees} Component={BasicFilters} />
-    <LinkButton onClick={() => setMode('builder')} style={{ marginTop: 15 }}>
-      Switch to Advanced Search Builder
-    </LinkButton>
+let PopOverItem = ({ children, ...props }) => (
+  <div style={{ margin: 10, cursor: 'pointer' }} {...props}>
+    {children}
   </div>
 )
 
+let BasicSearchFilters = ({ setMode, trees, children, BasicFilters }) => {
+  let [isOpen, setIsOpen] = React.useState(false)
+  return (
+    <div>
+      <Flex alignItems="center" justifyContent="space-between">
+        <Flex alignItems="center">
+          <h1>Filters</h1>
+        </Flex>
+        <div>
+          <Popover
+            isOpen={isOpen}
+            onClose={() => setIsOpen(false)}
+            style={{ width: 180, right: 0, top: 40 }}
+          >
+            <PopOverItem onClick={() => setMode('resultsOnly')}>
+              Toggle Filters
+            </PopOverItem>
+            <TreePauseButton children={children} Component={PopOverItem} />
+            <PopOverItem onClick={() => setMode('builder')}>
+              Advanced Search Builder
+            </PopOverItem>
+          </Popover>
+          <Icon
+            style={{ paddingTop: 15, cursor: 'pointer' }}
```

Comment:
same as above - we ideally wouldn't have this padding here

Reply:
We need the `cursor: 'pointer'` at least no?

Reply:
I used `DropdownItem` for the `clickable` thing as well ...